ozaria/site/common/ozariaUtils.js

```diff
@@ -30,18 +28,31 @@ export const findNextLevelsBySession = (sessions, levels, levelStatusMap) => {
     levelDataMap = levels
   }
   for (const [levelOriginal, level] of Object.entries(levelDataMap)) {
-    if (levelStatusMap[levelOriginal] === 'started') {
+    const levelStatus = levelStatusMap[levelOriginal]
+    const isLevelStarted = typeof levelStatus === 'string' && levelStatus === 'started'
+    const isLevelCompleted = (typeof levelStatus === 'string' && levelStatus === 'complete') || (typeof levelStatus === 'number' && levelStatus > 0)
+
+    if (isLevelStarted) {
       nextLevelOriginals.add(levelOriginal)
-    } else if (levelStatusMap[levelOriginal] === 'complete') {
-      const unlockedLevelOriginals = getNextLevelOriginalForLevel(level) || []
-      unlockedLevelOriginals
-        .filter((original) => levelStatusMap[original] !== 'complete')
-        .forEach((original) => nextLevelOriginals.add(original)) // incomplete unlocks
+    } else if (isLevelCompleted) {
+      let unlockedLevel = []
```

Comment:
Initialized here as an array. Used below (line 45) as an object. Should this be initialized as an empty object?
https://github.com/codecombat/codecombat/pull/5396/files#diff-351e4e90af49581dac77506a69dd3756R45

Reply:
Yep good catch, will update this to be an empty object.